server/service/transactions.service.js

```diff
@@ -0,0 +1,17 @@
+import Transaction from './schema/transaction.schema.js';
+
+export const getTransactions = async owner => await Transaction.find({ owner });
+
+export const createTransaction = async (date, year, month, type, category, comment, sum, owner) =>
+  await Transaction.create({ date, year, month, type, category, comment, sum, owner });
+
+export const updateTransactionById = async (id, field) =>
```

Comment:
Hmm zastanawia mnie:
Co, jeżeli użytkownik zmieni nie tylko, przypuśćmy kwotę, ale i kategorię. Na froncie otworzy sobie modal z edycją transakcji, zmieni dwa pola i jakiej funkcji użyjemy? może tu trzeba dać całkowitą aktualizację transakcji? Otwiera się okno edycji: pojawiają się poprzednio zdefiniowane wartości pól, i jak użytkownik je zmodyfikuje, to hurtem (jedną funkcję) uaktualnimy całą transakcję, sensownie?

Reply:
W paczkę field możemy wpakować w controllerze, co nam się podoba, może to być cały obiekt ze wszystkimi danymi z modala

Reply:
Analizując kod już mam dylemat: z nazwy field wykminiłem, że dotyczy to jednego pola do aktualizacji. Przychodzą mi do głowy dwa sposoby: 1. Można dać fields / allFields - z jednej strony dobrze, bo mamy załatwione "wszystkie" pola, które aktualnie są w schemacie transakcji. Lub 2. Dajemy konkretne pola ze schematy? Co Ty na to?

Reply:
Done